sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -446,7 +446,7 @@ def examples_wordcount_minimal(renames):
       # [END examples_wordcount_minimal_count]
 
       # [START examples_wordcount_minimal_map]
-      | beam.Map(lambda word_count: '%s: %s' % (word_count[0], word_count[1]))
```

Comment:
I saw such pattern in quite a few places so this may be a helpful syntactic sugar, as long as it does not make things difficult for us. 
I wouldn't rush this PR to 2.15.0 if we need more time to polish things.

Reply:
I don't think it'll make it harder for us, it's similar to the stuff we're already doing to support Map, Filter, etc. As for 2.15, I'm hoping we can start pushing Python 3 even more strongly then, which is why it'd be nice to get this syntactic sugar in.

Reply:
Ok, we should tag associated Jira with 'fix_version==2.15.0' in this case.